open-api/rest-catalog-open-api.py

```diff
@@ -852,6 +881,29 @@ class ViewRequirement(BaseModel):
     __root__: AssertViewUUID = Field(..., discriminator='type')
 
 
+class FailedPlanningResult(IcebergErrorResponse):
+    """
+    Failed server-side planning result
+    """
+
+    status: Literal['failed']
+
+
+class AsyncPlanningResult(BaseModel):
+    status: Literal['submitted']
+    plan_id: Optional[str] = Field(
+        None, alias='plan-id', description='ID used to track a planning request'
+    )
+
+
+class EmptyResult(BaseModel):
```

Comment:
```suggestion
class EmptyPlanningResult(BaseModel):
```

To be consistent with other results.

Reply:
Sorry, should have commented on the spec (I'll resolve this)